src/main/kotlin/us/spaceclouds42/ekho/Ekho.kt

```diff
@@ -46,13 +46,25 @@ class EkhoBuilder(base: LiteralText, method: EkhoBuilder.() -> Unit) {
         }
     }
 
+operator fun Text.invoke(inheritStyle: Boolean = true, method: EkhoBuilder.() -> Unit = { }) {
+        inherit = inheritStyle
+        if (method == { }) {
+this.let { it.style = root.style; siblings.add(it) }
+        } else {
+            siblings.add(EkhoBuilder(
+this.let { if (inheritStyle) { it.style = root.style }; it },
+                method
+            ).create())
+        }
+    }
+
     fun style(method: StyleBuilder.() -> Unit) {
         root.style = StyleBuilder(root.style.let { if (inherit) { it } else { Style.EMPTY } }).apply(method).create()
     }
 }
 
 class StyleBuilder(private val parentStyle: Style) {
-    private var color: TextColor? = null
+    var color: TextColor? = null
```

Comment:
```suggestion
    private var color: TextColor? = null
```
(not in scope of this change, also am going to rework color/style system a bit)